plugins/recruiting/src/components/NewCandidate.svelte

```diff
@@ -84,14 +92,15 @@ limitations under the License.
 </script>
 
 <div class="root">
-  <div class="header">
-    <div class="description">Add Candidate</div>
-    <div on:click={() => dispatch('close')}>
-      <Icon icon={workbench.icon.Close} button={true} />
-    </div>
-  </div>
   <ScrollView height="500px">
     <div class="form">
+      {#if spaces && spaceItems && spaceItems.length > 0}
+        <ComboBox label="Vacancy" items={spaceItems} bind:selected={selectedSpaceIdx}>
```

Comment:
Should we have a SpaceCombo component with a spaces and selected space passed, should be used in multiple places?